contracts/ibc-forwarder/src/msg.rs

```diff
@@ -119,6 +119,8 @@ pub enum MigrateMsg {
         clock_addr: Option<String>,
         next_contract: Option<String>,
         remote_chain_info: Option<RemoteChainInfo>,
+        // TODO: allow updating the transfer amount?
+        // transfer_amount: Option<Uint128>,
```

Comment:
I think this is reasonable but somehow should be done by also updating the rest of covenant fields that would be impacted

Reply:
lets make this updatable